src/main/java/report/ReportLong.java

```diff
@@ -0,0 +1,109 @@
+package report;
+
+import logic.CoursesStatus;
+import studentInfo.Student;
+import taskItems.Course;
+import taskItems.TrainingCenterTimetable;
+
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+import java.util.Formatter;
+
+public class ReportLong implements ITestReport{
+
+    private static int MIN_WIDE_REPORT = 32;
+    private static int LENGTH_WORD_CURRENT_DATE = 15;
+    private static int LENGTH_HOURS_COURSE = 4;
+    private static int LENGTH_NUMBER_COURSE = 3;
```

Comment:
if this is a constant than `final` is missing

Reply:
done